### DISCUSSION TITLE
Hybrid conference format dates, style, sites, etc.

### DISCUSSION THREAD

**Discussion by hidyverse** [resbazaz/festival2024#6] [Polls]
Hello ResBaz Festival 2024 Organizers,

Hurray! We've landed on a **hybrid conference** again for this year. 
    
I'm providing a starting point for a conference format based on historical conferences and ongoing discussions. As always, your discussion is encouraged. Because we need to agree on conference formatting before most planning tasks can be accomplished, **please provide all input on this matter by morning on Friday Feb 23**.  

For any answer that isn't "Yes", please provide discussion for your response. 
    
     ResBaz Festival 2024:
     3 Days  
     Wednesday - Friday, May 15-17
     2 Days In-person: 1 day unconference (Friday?), 1 day conference (Wednesday?), Talks given inside and outside
     1 Day entirely virtual (Thursday?)
     Will make strong effort to include multiple sites for virtual day and social event

**Comment by lwdozal** [comment]
Not sure if this matters anymore but WiDS Tucson is happening March 29th this year, which is potentially good for the date this poll proposes.

**Comment by Aariq** [comment]
I like this and it tempts me to lead both an unconference _and_ maybe do a talk or workshop on the virtual day

**Comment by brandon-jernigan** [comment]
I like it!! Just thinking any potential outdoor activities will be more comfortable in April

**Reply by hidyverse** [reply]
@Brandon I hear ya. I think you're in charge of logistics at this time so you can see to it we stay cool outdoors! Maybe that downstairs-outdoors part of Main Lib would be cool ?

**Comment by hidyverse** [comment]
We have agreed on the proposed conference format.
I'm closing this poll.
